includes/wikia/tests/UserTest.php

```diff
@@ -1,24 +1,144 @@
 <?php
+
+use Wikia\DependencyInjection\Injector;
+use Wikia\DependencyInjection\InjectorBuilder;
+use Wikia\Service\User\Preferences\PreferenceService;
+use Wikia\Service\User\Attributes\AttributeService;
+use Wikia\Domain\User\Preferences\UserPreferences;
+use Wikia\Service\User\Preferences\Migration\PreferenceScopeService;
+use Wikia\Service\User\Preferences\Migration\PreferenceCorrectionService;
+
+
 class UserTest extends WikiaBaseTest {
 
+	const TEST_USER_ID = 5;
+
+	protected $injector;
+	protected $userPreferenceServiceMock;
+	protected $userAttributeServiceMock;
+
+	protected $testUser;
+
+	public function setUp() {
+		parent::setUp();
+		$this->setupAndInjectUserPreferenceServiceMock();
+
+		$this->testUser = User::newFromId( self::TEST_USER_ID );
+	}
+
+	private function setupAndInjectUserPreferenceServiceMock() {
+		global $wgGlobalUserPreferenceWhiteList, $wgLocalUserPreferenceWhiteList;
+
+		$this->userPreferenceServiceMock = $this->getMock( 'Wikia\Service\User\Preferences\PreferenceService',
```

Comment:
It'd probably be better to use `PreferenceService::class` here, that way if we ever rename the interface IDEs could find usages of it here